museum/sampleRecords/donors.destination.json

```diff
@@ -0,0 +1,578 @@
+{
+  "ok":true,
+  "results":[
+    {
+      "entity_id":"1",
+      "id":"1",
+      "idno":"1",
+      "display_label":"N\/A",
+      "ca_entities.idno":"1",
+      "ca_entities.preferred_labels.displayname":"N\/A",
+      "ca_entities.preferred_labels.surname":"&lt;&gt;"
+    },
+    {
+      "entity_id":"2",
+      "id":"2",
+      "idno":"2",
+      "display_label":"HOPE A.T.",
+      "ca_entities.idno":"2",
+      "ca_entities.preferred_labels.displayname":"HOPE A.T.",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.forename":"A.T.",
+      "ca_entities.preferred_labels.middlename":"A.T.",
```

Comment:
I can't remember what we discussed at the meeting but `forename` and `middlename` are duplicated.  Is that a separate task to fix that issue?